app/views/sessions/authenticate.html.erb

```diff
@@ -1,74 +1,46 @@
-<% @parent_col_id = "signin" %>
-
-<%= page_heading (t :".page_heading"), sub_heading_text: (t :".page_sub_heading"), center: true %>
-
 <% login_info = get_login_info %>
 
-<h3>
-  Hi<%= login_info[:names].size == 1 ? " #{login_info[:names].first}!" : "!" %>
-</h3>
+<div class="ox-card login">
 
-<div id="options" class="row">
-  <div class="col-sm-12">
+  <h1 class="title"><%= t '.name_greeting', name: login_info[:names].first %></h1>
 
-    <% if login_info[:providers].include?('identity') %>
-      <div class="button-form-group">
-        <%= form_tag '/auth/identity/callback' do %>
-          <input type='hidden' id='auth_key' name='auth_key' class="form-control"
-                 value='<%= login_info[:username_or_email] %>'/>
-          <div class="form-group">
-            <label for='password'><%= t :".password" %></label>
-            <input type='password' id='password' name='password' class="form-control"/>
-          </div>
+  <% if login_info[:providers].include?('google_oauth2') %>
+    <%= link_to "/auth/google_oauth2", class: 'btn btn-social btn-google',
+                                       id: 'google-login-button' do %>
+      <span class="fa fa-google"></span> <%= t :".sign_in_with_google" %>
+    <% end %>
+  <% end %>
```

Comment:
Note: I removed twitter.  I **think** that was the plan?  Easy enough to restore if we want to.